ethportal-peertest/src/jsonrpc.rs

```diff
@@ -67,13 +80,25 @@ fn validate_endpoint_response(method: &str, result: &Value) {
                 ENR_SEQ.to_string()
             );
         }
+        "portal_stateFindNodes" => {
+            assert_eq!(
+                result.get("total").unwrap().as_u64().unwrap(),
+                peertest_nodes.nodes.len() as u64
+            );
```

Comment:
Wouldn't the total be equal to the number of `Nodes` messages, and not the number of nodes launched by the test?

Reply:
Ahh, good catch, I misinterpreted the spec there